### PR TITLE
Add hold configurator

### LINKED ISSUE #52: Front | Hold and bulkheads
Configurator for hold cargos:
- [x] edit hold cargos using table;
- [x] shiftable column for hold cargo with bool value;
- [x] configuration of bulkhead installed position;

### PR DESCRIPTION
**Pull request by nyaneet**
Closes #52

### CODE REVIEW

**Review by a-givertzman — CHANGES_REQUESTED** [review]
Check some comments in review about code quality, briefly:
- Try to fix / extend meaningless doc-comments
- try to avoid code duplication
- try to move/have functionality such as parsing `json` into the corresponding object
- try to avoid literals sparsed in whole code
- try ti decrease number of public methods
In generally - nice huge work, thank you!

File: lib/core/models/bulkheads/pg_bulkhead_places.dart [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                bp.bulkhead_id AS "bulkheadId",
+                bp.hold_group_id AS "holdGroupId"
+            FROM
+                bulkhead_place AS bp
+            WHERE
+                bp.ship_id = 1
+            ORDER BY "id" DESC;
+            """,
+      ),
+      entryBuilder: (row) => JsonBulkheadPlace(
```
[the remark this answers is not in the document]
Comment (a-givertzman) [reply lib/core/models/bulkheads/pg_bulkhead_places.dart:49]: discussed

File: lib/core/models/field/field_type.dart [hunk 2]
```diff
@@ -0,0 +1,22 @@
+///
+/// Enum of FieldData value types
```
[the remark this answers is not in the document]
Comment (nyaneet) [reply lib/core/models/field/field_type.dart]: Fixed in b71e9effe7824a166fda63e66ae45bbd09bed226

File: lib/core/models/bulkheads/json_bulkhead.dart [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import 'package:sss_computing_client/core/models/bulkheads/bulkhead.dart';
+///
+/// [Bulkhead] that parses itself from json map.
+class JsonBulkhead implements Bulkhead {
+  final Map<String, dynamic> _json;
+  ///
+  /// Creates [Bulkhead] that parses itself from provided `json` map.
```
[the remark this answers is not in the document]
Comment (nyaneet) [reply lib/core/models/bulkheads/json_bulkhead.dart]: Fixed in fdec20a4aa264faef2d8f63a21f3c7a6951bc9cd for whole project
It turned out to be easy, based on regular expressions and vs code replace feature (e.g. ``` `([a-zA-Z]) ``` in search pattern and ``` $1[ ``` in replace expression for ``` ` ``` symbol followed by any letter)

Correctness of fix verified based on [Effective Dart: Documentation](https://dart.dev/effective-dart/documentation#do-use-square-brackets-in-doc-comments-to-refer-to-in-scope-identifiers)

**Comment by nyaneet** [comment]
Added documentation as submodule in 93bab0e30b9e73bbd52d181dac5ba7c10d21c43c

**Review by a-givertzman — DISMISSED** [review]

**Pull request merged**